src/cellEditors/CellEditor.js

```diff
@@ -242,8 +233,6 @@ var CellEditor = Base.extend('CellEditor', {
             this.grid.cellEditor = null;
             this.el.remove();
         } else if (feedback >= 0) { // never true when `feedback` undefined
-            var point = this.event.gridCell;
-            this.grid.selectViewportCell(point.x, point.y - this.grid.getHeaderRowCount());
```

Comment:
The intent of the deleted line in question is to reposition the edit box before giving an error animation + alert. Not sure why that was necessary but seems to work without it. More to the point, that function `selectViewportCell ` is broken and anyway and needs to be fixed. We can do that later. This should be good to go as is.